### ISSUE TITLE
Deprecation warning due to invalid escape sequences in Python 3.8

### ISSUE THREAD

**Issue by tirkarthi** [rollbar/pyrollbar#327]
Deprecation warnings are raised due to invalid escape sequences in Python 3.8 . Below is a log of the warnings raised during compiling all the python files. Using raw strings or escaping them will fix this issue.

```
find . -iname '*.py'  | xargs -P 4 -I{} python -Walways -m py_compile {}

./rollbar/test/test_rollbar.py:685: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals'][varargs][0], '\*+')
./rollbar/test/test_rollbar.py:712: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals'][varargs][0], '\*+')
./rollbar/test/test_rollbar.py:713: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals'][varargs][1], '\*+')
./rollbar/test/test_rollbar.py:876: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals']['password'], '\*+')
./rollbar/test/test_rollbar.py:930: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals'][keywords]['password'], '\*+')
./rollbar/test/test_rollbar.py:961: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals']['password'], '\*+')
./rollbar/test/test_rollbar.py:962: DeprecationWarning: invalid escape sequence \*
  self.assertRegex(payload['data']['body']['trace']['frames'][-1]['locals']['Password'], '\*+')
./rollbar/contrib/django/middleware.py:1: DeprecationWarning: invalid escape sequence \.
  """
```

**Comment by Alfor89** [comment]
#

**Comment by terencehonles** [comment]
Fixed by #331

**Comment by mrunalk** [comment]
Closing it as it seems to be fixed.
